setup/setup_rirs.py

```
@@ -0,0 +1,406 @@
+            "file_type": "zip"
+        },
+        "openair": {
+            "name": "OpenAIR Impulse Responses",
+            "url": "https://www.openslr.org/resources/22/openair-impulse-responses.zip",
+            "description": "Various room impulse responses from OpenAIR",
+            "size_mb": 50,
+            "file_type": "zip"
+        }
+    }
+
+    def download_file(self, url: str, destination: Path, chunk_size: int = 8192) -> bool:
```

Ruff (0.13.1)

Unused method argument: `chunk_size`

(ARG002)